### PR TITLE
Configure Firebase

### PR DESCRIPTION
**Pull request by zghera**
Add configuration for using Firebase and its related products (Firestore and Firebase Auth in our case).

Removed all starter portfolio code from index.html, style.css, and script.js altogether. Also created scripts and styles folder to match structure in [design doc](https://docs.google.com/document/d/1ogEBq2Bssc6VrcoCO5gpw3v2dSc3Zd-f9Y4C6HNFLNg/edit#).

### CODE REVIEW

File: slurp/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,21 @@
...
+    <title>Slurp</title>
+    <!-- CSS -->
+    <link rel="stylesheet" href="/styles/style.css">
+
+    <!-- JavaScript -->
+    <!-- The core Firebase JS SDK is always required and must be listed first -->
+    <script defer src="https://www.gstatic.com/firebasejs/7.15.4/firebase-app.js"></script>
+    <!-- Add any other SDKs for Firebase products that we want to use
+        https://firebase.google.com/docs/web/setup#available-libraries -->
+    <script defersrc="https://www.gstatic.com/firebasejs/7.15.4/firebase-firestore.js"></script>
```
Comment (keiffer01) [inline slurp/src/main/webapp/index.html:14]:
```suggestion
    <script defer src="https://www.gstatic.com/firebasejs/7.15.4/firebase-firestore.js"></script>
```

File: slurp/src/main/webapp/scripts/init-firebase.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// Initialize Firebase and its related product we use (Firestore and Firebase
+// Auth) using the provided Firebase project configuation.
```
Comment (keiffer01) [inline slurp/src/main/webapp/scripts/init-firebase.js:17]: Although unlikely in our current plans that we'll use any other Firebase products, maybe remove the "(Firestore and Firebase Auth)" to keep it general?

File: slurp/src/main/webapp/index.html (hunk 1 above)
Comment (zghera) [reply slurp/src/main/webapp/index.html]: Done.

File: slurp/src/main/webapp/scripts/init-firebase.js (hunk 2 above)
Comment (zghera) [reply slurp/src/main/webapp/scripts/init-firebase.js]: Good point. Done.

File: slurp/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,21 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <title>Slurp</title>
+    <!-- CSS -->
+    <link rel="stylesheet" href="/styles/style.css">
```
Comment (ananyeet) [inline slurp/src/main/webapp/index.html:7]:
```suggestion
    <link rel="stylesheet" href="styles/style.css">
```

These links should be relative.

File: slurp/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,21 @@
...
+    <!-- JavaScript -->
+    <!-- The core Firebase JS SDK is always required and must be listed first -->
+    <script defer src="https://www.gstatic.com/firebasejs/7.15.4/firebase-app.js"></script>
+    <!-- Add any other SDKs for Firebase products that we want to use
+        https://firebase.google.com/docs/web/setup#available-libraries -->
+    <script defer src="https://www.gstatic.com/firebasejs/7.15.4/firebase-firestore.js"></script>
+    <script defer src="https://www.gstatic.com/firebasejs/7.15.4/firebase-auth.js"></script>
+
+    <script defer src="/scripts/init-firebase.js"></script>
+    <script defer src="/scripts/script.js"></script>
```
Comment (ananyeet) [inline slurp/src/main/webapp/index.html:18]:
```suggestion
    <script defer src="scripts/init-firebase.js"></script>
    <script defer src="scripts/script.js"></script>
```

These links should be relative.

File: slurp/src/main/webapp/index.html (hunk 3 above)
Comment (zghera) [reply slurp/src/main/webapp/index.html]: Sure. I am curious why though since they have the same effect?

**Pull request merged**